pyproject.toml

```diff
@@ -10,4 +10,5 @@ build-backend = 'setuptools.build_meta'
 
 [tool.astropy-bot.autolabel]
 
+# Comment this out to re-enable but then labeler Action needs to be disabled.
```

Comment:
I think the comment out way would only work if the default is `False`. My problem was in the other PR, is that if the default is True, and there is only string bool eval, then the `'false'` below would be a `True`, and thus there is no way to disable from the default.

Reply:
I am not familiar with TOML parsing. I thought a "false" would be parsed as `False` (boolean)? I am following some other existing code. I am confused now.

Reply:
ah, ok, that parsing is a possibility. Either case, as you said it all followed the existing code (for which I didn't see much testing, thus couldn't double checked it, but that's an issue for the bot/baldrick).

Reply:
For future reference: https://github.com/astropy/astropy-bot/pull/123#issuecomment-723130381 (I think the config is correct.)